pkg/console/console.go

```diff
@@ -53,6 +54,7 @@ func (s StandardConsole) Run(cmd string, opts ...func(cmd *exec.Cmd)) (string, e
 	for _, o := range opts {
 		o(c)
 	}
+	displayProgress(s.logger, 10*time.Second, fmt.Sprintf("Still running command '%s'", cmd))
```

Comment:
uhmmmm. The channel that is returned here is not closed, I guess that would display forever until we close it?

Reply:
Somehow it doesnt?

Reply:
how come :D

Reply:
ahh I guess becuase once its done, it returns and the full context gets destroyed when the function ends by the garbage collector, so no more messages?

Reply:
mmm wouldn't make sense, the goroutine should be still there ticking.. no?

Reply:
yes, this is c orrect but will only show if a later command takes long :D

Reply:
yep, now fixed!